File: WordPrefix/isPrefixOfWord.py
def isPrefixOfWord(sentence: str, searchWord: str) -> int:
    words = sentence.split(" ")
    for i, word in enumerate(words):
        if word.startswith(searchWord):
            return i + 1
    return -1

print(isPrefixOfWord("i love eating burger", "burg"))

File: WordPrefix/IsPrefixOfWord.java
class Dukwon {
    public static int isPrefixOfWord(String sentence, String searchWord) {
        String[] splitStr = sentence.split(" ");
        for (int i = 0; i < splitStr.length; i++) {
            if (splitStr[i].startsWith(searchWord)) {
                return i + 1;
            }
        }
        return -1;
    }
    public static void main(String[] args) {
        System.out.println(isPrefixOfWord("i love eating burger", "burg"));
    }
}